src/modules/esl-tab/core/esl-tabs.ts

```diff
@@ -72,8 +87,8 @@ export class ESLTabs extends ESLBaseElement {
   }
 
   /** Active {@link ESLTab} item */
-  public get $current(): ESLTab | null {
-    return this.$tabs.find((el) => el.active) || null;
+  public get $current(): ESLTab | undefined {
```

Comment:
Do we really need that change?
`undefined` is the value that is similar to the unsupported key, while the null value indicates that it is supported key but there is no current tub right now